src/pages/projectSettings/tabs/NotificationsTab/SubscriptionField.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from "@emotion/styled";
+import ExpandableCard from "@leafygreen-ui/expandable-card";
+import { Body } from "@leafygreen-ui/typography";
+
+import { Field } from "@rjsf/core";
+import { size } from "constants/tokens";
+import { triggers } from "constants/triggers";
+import { Subscriber } from "gql/generated/types";
+import { ResourceType } from "types/triggers";
+import { toSentenceCase } from "utils/string";
+import { SubscriptionSelect } from "./NotificationsRow";
+
+export const getSubscriberType = (subscriber: Subscriber) => {
+  if (subscriber === undefined) {
+    return "";
+  }
+  let key: string = "";
+  Object.keys(subscriber).forEach((k) => {
+    if (subscriber[k] !== null) {
+      key = k;
+    }
+  });
+  return key;
+};
+
+export const getTrigger = (trigger: string, resourceType: string) => {
```

Comment:
nit. add return type `Trigger` since it's not immediately obvious what's being returned